gateway-service-impl/build.gradle.kts

```diff
@@ -28,12 +34,19 @@ dependencies {
   // Metrics
   implementation("io.dropwizard.metrics:metrics-core:4.1.9")
 
+  runtimeOnly("io.netty:netty-codec-http2:4.1.53.Final") {
```

Comment:
Does the impl use netty or is it only needed by the service (which also declares this)?

Reply:
I don't think impl should use netty but I was surprised to see this failure where impl module was reporting Snyk failure in these libraries. https://app.circleci.com/pipelines/github/hypertrace/gateway-service/195/workflows/528c60c9-0589-49e5-848d-9f486838ed25/jobs/564/parallel-runs/0/steps/0-106

I don't see that dependency coming from anywhere else too. Did you see such behavior?

Reply:
I haven't run into that in an impl package. I checked, netty is not part of the impl's runtime classpath. I'd guess line 51 triggered it since I believe snyk scans all configurations by default. So I'd suggest two things: change 51 to `testRuntimeOnly` (if it's actually needed, most unit tests don't unless you're actually spinning up a local channel which is usually not a great idea) , and move the constraints to testRuntimeOnly as well.

Reply:
Thanks. That seems to be the case. i've fixed this now.